beacon_node/network/src/sync/block_lookups/common.rs

```diff
@@ -371,7 +371,7 @@ impl<L: Lookup, T: BeaconChainTypes> RequestState<L, T> for BlobRequestState<L,
 
     fn verify_response_inner(
         &mut self,
-        _expected_block_root: Hash256,
+        expected_block_root: Hash256,
```

Comment:
I add this in the other PR but it's no longer needed here:

```suggestion
        _expected_block_root: Hash256,
```